text/java/0000-clojure-tools.md

```diff
@@ -0,0 +1,45 @@
+# Clojure Tools Buildpack
+
+## Summary
+
+A Clojure Tools buildpack for building Clojure applications using [deps][d] and [tools build][t] has been created [by the community][b].  It is based on the same technologies and is nearly indistinguishable from the other Paketo Buildpacks, especially the JVM build-system buildpacks, and it should be adopted into the project.
+
+[d]: https://clojure.org/guides/deps_and_cli
+[t]: https://clojure.org/guides/tools_build
+[b]: https://github.com/eddumelendez/clojure
+
+## Motivation
+
+This is an additional buildpack, for building JVM-based applications based on Clojure, and offers yet another workload that can be built by Paketo buildpacks.
+
+## Detailed Explanation
+
+The Clojure Tools Buildpack will be moved into the paketo-buildpacks Github organization and added to the Java sub-team. Maintenance for this buildpack will be provided by the Java subteam.
+
+The following changes will be made:
+
+* [Clojure Tools Buildpack][b] moved to Paketo Buildpacks org
+* Buildpack will have paketo-buildpacks/clojure ID
+* Buildpack will be published to gcr.io/paketo-buildpacks/clojure
```

Comment:
```suggestion
* Buildpack will have paketo-buildpacks/clojure-tools ID
* Buildpack will be published to gcr.io/paketo-buildpacks/clojure-tools
```

Maybe `clojure-tools` is more explicit? It, doesn't seem like a common use case but one could theoretically [build a clojure project with maven](https://github.com/talios/clojure-maven-plugin) or another tool chain.